pkgs/shells/zsh/zsh-edit/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-edit";
+  version = "unstable-2022-05-05";
+
+  src = fetchFromGitHub {
+    owner = "marlonrichert";
+    repo = "zsh-edit";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
we shouldn't replace every string that happens to match pname

Reply:
sorry, I thought there was some desire to keep these configs DRY.

also, should we not push to exact context linked strings replaced, or since they are largely unchanging it does not matter?